### PR TITLE
reducing night length from 25>20

### LINKED ISSUE #78: Shorten maximum night length
Currently nights last for a random number between 5 and 25 turns. If you are doing a lot on your turn, this can result in a long time between climatic changes.

**Acceptance Criteria:**
- Reduce the maximum night length from 25 to 20 turns.

### PR DESCRIPTION
**Pull request by sculley**
Shortening the maximum number of nights from 25>20

Fixes #78

### CODE REVIEW

File: game.py [hunk 1]
```diff
@@ -611,7 +611,7 @@ def end_turn(self) -> bool:
                 if self.until_night == 0:
                     self.board.overlay.toggle_night(True)
                     # Nights last for between 5 and 25 turns.
```
Comment (ChrisNeedham24) [inline game.py:613]: Sorry to nitpick, but you'll also need to update the comment here.

**Review by ChrisNeedham24 — CHANGES_REQUESTED** [review]
See the above thread.

File: game.py (hunk 1 above)
Comment (sculley) [reply game.py]: Of course, thanks for spotting that one!

**Comment by sculley** [comment]
Comment addressed, I have squashed my commits back to a single one.

**Review by ChrisNeedham24 — APPROVED** [review]
LGTM

**Pull request merged**